src/_P036_FrameOLED.ino

```diff
@@ -1357,22 +1360,22 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 # ifdef PLUGIN_036_DEBUG
 
           if (eventId == P036_EVENT_LINE) {
-          String log;
+            String log;
 
-          if (loglevelActiveFor(LOG_LEVEL_INFO) &&
-              log.reserve(200)) { // estimated
+            if (loglevelActiveFor(LOG_LEVEL_INFO) &&
+                log.reserve(200)) { // estimated
               log  = F("[P36] Line: ");
-            log += LineNo;
-            log += F(" Content:");
+              log += LineNo;
+              log += F(" Content:");
               log += P036_data->LineContent->DisplayLinesV1[LineNo - 1].Content;
-            log += F(" Length:");
+              log += F(" Length:");
               log += P036_data->LineContent->DisplayLinesV1[LineNo - 1].Content.length();
-            log += F(" Pix: ");
+              log += F(" Pix: ");
               log += P036_data->display->getStringWidth(P036_data->LineContent->DisplayLinesV1[LineNo - 1].Content);
-            log += F(" Reserved:");
+              log += F(" Reserved:");
               log += P036_data->LineContent->DisplayLinesV1[LineNo - 1].reserved;
-            addLogMove(LOG_LEVEL_INFO, log);
-              delay(5); // otherwise it is may be to fast for the serial monitor
+              addLogMove(LOG_LEVEL_INFO, log);
+              delay(5); // FIXME otherwise it is maybe too fast for the serial monitor
```

Comment:
These aweful delay() calls should be in #ifdef wrappers.
I'm working so hard to get the code running as smooth as I can and I will not tolerate delays in log calls, especially when they are LOG_LEVEL_INFO.
If the log can't keep up, then why should we create such long debug like logs at loglevel info?

Reply:
They are in ifdefs for development logging, not supposed to ever be enabled for production use